_topic_maps/_topic_map.yml

```diff
@@ -840,6 +840,17 @@ Topics:
     File: file-integrity-operator-advanced-usage
   - Name: Troubleshooting the File Integrity Operator
     File: file-integrity-operator-troubleshooting
+- Name: cert-manager Operator for Red Hat OpenShift
+  Dir: cert_manager_operator
+  Topics:
+  - Name: About the cert-manager Operator for Red Hat OpenShift
+    File: index
+  - Name: cert-manager Operator for Red Hat OpenShift release notes
+    File: cert-manager-operator-release-notes
+  - Name: Installing the cert-manager Operator for Red Hat OpenShift
+    File: cert-manager-operator-install
+  - Name: Uninstalling the cert-manager Operator for Red Hat OpenShift
```

Comment:
@slaskawi So right now the docs go from installing the operator to uninstalling the operator.

I know you mentioned that you're doing any config in the unsupported section of the CRD, so that won't get put into the docs.

But is there something else here that we should cover on actually using it? I see in the TE slides that you have a demo for creating a certificate, but wasn't sure if that was something we should show here or not. And if so, can you help me with what the steps/details would be?

Reply:
No, we don't do anything special for uninstalling at the moment. 

Possibly, in the future we might want to wipe the operand our as well. But that's for the future.

Reply:
@slaskawi Apologies, I don't think I wrote out exactly what I meant. I'm not asking about uninstalling here. I'm asking whether we need a section after you install the operator to tell users what they actually do next to actually use cert-manager. Like in your TE slides, you mention creating a cert in your demo after you installed the operator.

Reply:
@slaskawi After thinking about it more, I think that this is the part we said we might wait til the GA release of cert-manager to cover. So I won't worry about this yet.

Reply:
Yes, totally agree.